From: Valentine Fatiev <valentinef@nvidia.com>
Subject: [PATCH] BACKPORT:
 drivers/net/ethernet/mellanox/mlx5/core/en/tc/post_meter.h

Change-Id: Ia4c4525983ed7133b18ebf7b5347c2b375b7ef37
---
 drivers/net/ethernet/mellanox/mlx5/core/en/tc/post_meter.h | 1 +
 1 file changed, 1 insertion(+)

--- a/drivers/net/ethernet/mellanox/mlx5/core/en/tc/post_meter.h
+++ b/drivers/net/ethernet/mellanox/mlx5/core/en/tc/post_meter.h
@@ -4,6 +4,7 @@
 #ifndef __MLX5_EN_POST_METER_H__
 #define __MLX5_EN_POST_METER_H__
 
+
 #define packet_color_to_reg { \
 	.mfield = MLX5_ACTION_IN_FIELD_METADATA_REG_C_5, \
 	.moffset = 0, \
